### ISSUE TITLE
Ignore issue

### ISSUE THREAD

**Issue by jtrakk** [julia-vscode/StaticLint.jl#321]
Sometimes StaticLint incorrectly reports an issue and I want to ignore it. I want to comment e.g.

```jl
f(1,2,3) # staticlint-ignore: methodcall(f)
```
